Jenkinsfile

```diff
@@ -0,0 +1 @@
+buildPlugin()
```

Comment:
Nice2have: line break.
Ideally it should have been in a separate pull request, but it's fine to keep it there IMHO (not a bugfix)

Reply:
Line break added

Reply:
JFYI: text files MUST end with line breaks for POSIX definitions.

* http://pubs.opengroup.org/onlinepubs/9699919799/basedefs/V1_chap03.html#tag_03_403
* http://pubs.opengroup.org/onlinepubs/9699919799/basedefs/V1_chap03.html#tag_03_206